packages/create-mc-app/src/validations.ts

```diff
@@ -87,11 +88,24 @@ const throwIfNodeVersionIsNotSupported = (
   }
 };
 
+const throwIfCloudRegionIsNotSupported = (cloudRegion: string) => {
+  const allowedCloudIdentifiers = Object.values(CLOUD_IDENTIFIERS) as string[];
+
+  if (!allowedCloudIdentifiers.includes(cloudRegion)) {
+    throw new Error(
+      `The cloud region "${cloudRegion}" is not supported. Supported regions are: ${Object.values(
+        CLOUD_IDENTIFIERS
+      ).join(', ')}.`
```

Comment:
nit:
```suggestion
      `The cloud region "${cloudRegion}" is not supported. Supported regions are: ${allowedCloudIdentifiers.join(', ')}.`
```

Reply:
Updated in this commit: [6c34369](https://github.com/commercetools/merchant-center-application-kit/pull/3564/commits/6c343695099ba5e75208ba31fa7ba2dfa63da7ff)